src/main/java/org/spongepowered/api/text/message/Message.java

```diff
@@ -221,21 +222,19 @@
      */
     interface Score extends Message {
 
-        // TODO use Score
-
         /**
-         * Returns the score displayed by this {@link Message}.
+         * Returns the {@link org.spongepowered.api.scoreboard.Score} displayed by this {@link Message}.
          *
-         * @return The score in this message
+         * @return The {@link org.spongepowered.api.scoreboard.Score} in this message
          */
         @Override
-        Object getContent();
+        org.spongepowered.api.scoreboard.Score getContent();
```

Comment:
I agree on that. Naming things the same may be good for mental association and shortness, but this will be totally break if you try to use them simultaneously.
(The Builder should be renamed then as well)

The same applies to Selector, but it should probably be fixed outside of this PR.